### ISSUE TITLE
Color tests & filler assets for levels

### ISSUE THREAD

**Issue by pushfoo** [Deli-Slicer/CatBurglar#9]
## tl;dr
We want to know if we can make the game work visually, so we need interactive mockups. Detail isn't a priority, but color language and palettes should be a little at this point. My hope is to end up with color language that's clear. [Mirror's Edge 1](https://www.worldofleveldesign.com/categories/game_environments_design/mirrors-edge-color.php) and the Portal games are top notch examples of good puzzle game level design for 1st Person. [Limbo](https://www.youtube.com/watch?v=Y4HSyVXKYz8) is a highly recommended side scroller puzzle game.

## Assets to try adding
Use the work from #8 to try building assets and a map that can be previewed in TMX with some of the following:
- [ ] Desks for NPCs facing the screen. these can be rectangles
- [ ] Chairs for NPCs facing the screen. 
- [ ] Coffee machine on a stand
- [ ] Water cooler
- [ ] Light switches ( these will be hard to make readable )
- [ ] Bathroom door
- [ ] Background tiles of various sorts.

It may should use the layers outlined in #3  as guidance for how to break things up, but you don't have to make it playable yet. If you make something that you think looks good, merge it in under a reasonable location. I'll try to get the layer manager / renderer working.

## Color palette suggestions
If these suggestions suck, feel free to throw them out. My initial color suggestions are as follows:
- Light grays, toward white, with a hint of blue for large areas of interaction: water cooler stands, doors, and any light switches
- Darker grays toward black for interaction hints: chairs, gorilla, and coffee maker's body
- Medium grays with a tiny hint of yellow for the background layer (not certain about this one)

I might be wrong about these colors . **You can draw and paint much better than I can so use your best judgement to come up with a police station color palette that seems reasonable**. The goal is a light-hearted tone, not a serious one.
